docs/management/advanced-options.asciidoc

```diff
@@ -586,8 +577,10 @@ The URL of a custom SVG image that appears on {kib} browser tabs. Images must be
 The URL of a custom PNG image that appears on {kib} browser tabs.
 
 [float]
-[[kibana-usage-data-settings]]
-==== Usage Data
+[[kibana-usage-collection-settings]]
+==== Usage Collection
```

Comment:
We already use `Usage Data`, `Usage Collection` and `Telemetry`. We want to remove `Usage Data` when possible